packages/svelte2tsx/src/htmlxtojsx_v2/nodes/AttachTag.ts

```diff
@@ -0,0 +1,15 @@
+import { BaseNode } from '../../interfaces';
+import { Element } from './Element';
+import { InlineComponent } from './InlineComponent';
+
+/**
+ * {@attach xxx}   --->    [Symbol()]: xxx
+ */
+export function handleAttachTag(tag: BaseNode, element: Element | InlineComponent): void {
+    // element.addAttachment(attr);
+    if (element instanceof InlineComponent) {
+        element.addProp(['[Symbol()]'], [[tag.expression.start, tag.expression.end]]);
```

Comment:
it expects `symbol.description === '@attach'`, just to give us some very basic filtering ability. not sure if it matters here (I assume it just needs to satisfy the index key constraint) but figured we may as well be thorough

```suggestion
        element.addProp(['[Symbol("@attach")]'], [[tag.expression.start, tag.expression.end]]);
```